index.bs

```diff
@@ -592,7 +592,6 @@ A [=sensor=] has an associated <dfn>current polling frequency</dfn> which is ini
 <pre class="idl">
 [SecureContext]
 interface Sensor : EventTarget {
-  readonly attribute SensorState state;
```

Comment:
I think we should just avoid exposing it for now. That's the kind of precise feedback I'd like to see from origin trials, though (maybe we could even expose both there and see which ones devs end up using?)